src/main/java/org/snomed/snowstorm/core/data/services/ConceptService.java

```diff
@@ -58,6 +61,8 @@
 @Service
 public class ConceptService extends ComponentService {
 	private static final Map<ComponentType, Class<? extends DomainEntity<?>>> COMPONENT_DOCUMENT_TYPES = new HashMap<>();
+	private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd")
```

Comment:
@kaicode , I'm always wary when it comes to using time in Java. Are you happy with my approach of using `DateTimeFormatter` and `Instant` ?

Reply:
Looks good to me. I still tend to use `SimpleDateFormat` but it's not thread safe. This Java 8 method looks better 👍

Reply:
I have removed this as the `codeSystemService.findAllVersions()` method call does indeed return the correct versions.